src/Orc.Controls/Controls/StepBar/Models/Interfaces/IStepBarItem.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Orc.Controls.Controls.StepBar.Models
+{
+    using System;
+    using Catel.MVVM;
+
+    public interface IStepBarItem
+    {
+        ISummaryItem GetSummary();
+
+        IViewModel ViewModel { get; set; }
+
+        event EventHandler<ViewModelChangedEventArgs> ViewModelChanged;
+
+        string Title { get; set; }
+
+        string BreadcrumbTitle { get; set; }
+
+        string Description { get; set; }
+
+        int Number { get; set; }
+
+        bool IsOptional { get; }
```

Comment:
I think we need to revise this into a state enum where we can have different states for a stepbar item (not linked to the wizard in any way)